docs/zilevm/dev-onboarding/remix.md

```diff
@@ -17,17 +17,17 @@ Once you have set up Metamask to have the Zilliqa EVM configuration, then open [
 
 Open the Workspaces folder and right click compile the file called `1_Storage.sol` which will build some artifacts.
 
-!["Compiling a contract"](/assets/img/evm/compile_contract.png)
+<img alt="Compiling a contract" src="/assets/img/evm/compile_contract.png" width="400px">
```

Comment:
Did you consider using the `attr_list` extension (already setup):
```md
![Compiling a contract](/assets/img/evm/compile_contract.png){: style="width:150px"}
```
I'll leave it to you whether you prefer HTML or Markdown.